src/segments/umbraco.go

```diff
@@ -0,0 +1,231 @@
+package segments
+
+import (
+	"encoding/xml"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/platform"
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+)
+
+type Umbraco struct {
+	props properties.Properties
+	env   platform.Environment
+
+	FoundUmbraco    bool
+	IsModernUmbraco bool
+	IsLegacyUmbraco bool
+	Version         string
+}
+
+const (
+	umbracoFolderName = "umbraco"
+	umbracoWebConfig  = "web.config"
+)
+
+type CSProj struct {
+	PackageReferences []struct {
+		Name    string `xml:"Include,attr"`
+		Version string `xml:"Version,attr"`
+	} `xml:"ItemGroup>PackageReference"`
+}
+
+type WebConfig struct {
+	AppSettings []struct {
+		Key   string `xml:"key,attr"` // TODO: What happens if the web.config has the attribute as uppercase Key="" ?
```

Comment:
You can easily unit test the question

Reply:
OK for future reference XML unmarshal is case sensitive, and was able to check clearly with the tests if I fudged/change the files.

This was fixed by simple string replacement to ensuring the XML attributes I am interested has been converted to lowecase